bin/run-detectors-timelines.sh

```diff
@@ -55,6 +61,26 @@ usage() {
     --focus-timelines   only produce the detector timelines, do not run detector QA code
     --focus-qa          only run the QA code (assumes you have detector timelines already)
 
+    --run-slurm         run timelines on SLURM instead of running multi-threaded locally
```

Comment:
Could you also update the documentation how to use these new options?
- `doc/chef_guide.md`: supposed to be as terse as possible
- `doc/procedure.md`: where you don't have to be terse (in fact, `chef_guide.md` was created because `procedure.md` was _too_ verbose...)